qiskit/transpiler/passes/optimization/optimize_1q_decomposition.py

```diff
@@ -147,62 +201,42 @@ def run(self, dag):
                 basis = self._target.operation_names_for_qargs((qubit,))
 
             if new_dag is not None and self._substitution_checks(dag, run, new_dag, basis, qubit):
+                new_dag = self._gate_sequence_to_dag(new_dag)
                 dag.substitute_node_with_dag(run[0], new_dag)
                 # Delete the other nodes in the run
                 for current_node in run[1:]:
                     dag.remove_op_node(current_node)
 
         return dag
 
+    def _error(self, circuit, qubit):
+        """
+        Calculate a rough error for a `circuit` that runs on a specific
+        `qubit` of `target` (circuit could also be a list of DAGNodes)
+
+        Use basis errors from target if available, otherwise use length
+        of circuit as a weak proxy for error.
+        """
```

Comment:
I think this comment is slightly out-of-date.  I think `circuit` is either a `DAGCircuit` or a `OneQubitGateSequence` (and `target` isn't passed any more).

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/9578/commits/89db1da2d797475094449d93a2e6eb043db58fae